src/components/alert-banner/alert-banner.js

```diff
@@ -44,30 +42,27 @@ export default function() {
   */
   function addAlertCookie(alert) {
     const cookieName = cookieBuilder.dataset(alert, 'cookie');
-    if (cookieName) {
+    if (cookieName)
       cookieBuilder.createCookie(
           cookieName,
           'dismissed',
           cookieBuilder.getDomain(window.location, false),
           360
       );
-    }
   }
 
   const alerts = document.querySelectorAll('.js-alert');
 
-  if (alerts.length) {
+  if (alerts.length)
     for (let i=0; i <= alert.length; i++) {
```

Comment:
@kimpenguin when run `npm run predeploy` I get ` 57:5  error  Unnecessary { after 'for' condition  curly`. Which comes from the for loop brackets `{}`. But when these are removed it creates another ESLint style violation. This is just Eslint style violation not an actual code error.